src/main/java/com/google/sps/data/PlayerDatabase.java

```diff
@@ -23,15 +23,23 @@
   private static final String LOGGED_OUT_EXCEPTION =
       "Player is currently logged out. Cannot process null user.";
   private static final Query query = new Query(ENTITY_QUERY_STRING);
-  private static final User USER = UserServiceFactory.getUserService().getCurrentUser();
-  private static boolean isLoggedIn = UserServiceFactory.getUserService().isUserLoggedIn();
+  private static User USER;
+  private static boolean isLoggedIn;
 
   public static List<Player> getPlayers() {
     return players;
   }
 
   public PlayerDatabase(DatastoreService datastore) {
```

Comment:
This constructor can call the next constructor:

this(datastore, UserServiceFactory.getUserService)